CHANGELOG.md

```diff
@@ -31,11 +31,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Removed obsoleted methods from 2.x
 - Removed mutable properties on OpenXmlAttribute and marked as `readonly`
 
+### Fixed
+- Fixed incorrect file extensions for vbaProject files (#1292)
+
 ## [2.20.0]
 
 ### Changed
 - Marked all obsoleted APIs as compile as error. These will be removed in the next release
 
+### Fixed
```

Comment:
nit: in the future, you don't need to worry about update 2.20 on main; we'll copy the lisitng from that branch over to here